app/assets/stylesheets/modules/mfa.css

```diff
@@ -7,6 +7,29 @@
   flex: 1;
 }
 
+.mfa__header-wrapper {
+  display: flex;
+  align-items: center;
+  gap: 12px;
```

Comment:
Do you feel that the header and the badge are a bit close? I feel like a 30px gap looks better
<img width="790" alt="Screenshot 2023-07-17 at 10 29 32 AM" src="https://github.com/rubygems/rubygems.org/assets/42748004/ee73622a-49eb-4898-92dd-dcc5c9df5ff7">

Alt I tried to align the badge by justifying the badge to the right but it feels like there's too much space?

```
justify-content: space-between;
```
<img width="778" alt="Screenshot 2023-07-17 at 10 29 51 AM" src="https://github.com/rubygems/rubygems.org/assets/42748004/449b751b-325e-45e6-a6de-87258ebd53b5">

Reply:
I personally like how it is right now but would be open to the 30px gap if others think that looks better (agreed on the right-justified looking slightly too jarring)

Reply:
cc: @bettymakes @Schwad @ericherscovich

Reply:
Sorry! Missed replying to this. Haha this one's tough too! A lot of this is us chiming in on our personal tastes. In the future, it'd be great to have patterns around spacing so we can establish vertical rhythm and density of content etc.

From eyeballin', I prefer the closer a bit more 😅. This isn't a make-it-or-break-it. The majority of y'all prefer `20px`, so stick with that 👍. 

-----

However, I'm pretty strongly opposed to the right-aligned badges. It makes it harder to scan the labels associated to its heading at a glance. People are lazy 😂 . Don't make their eyes do more work haha.